pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/DummyJavaNode.java

```diff
@@ -20,6 +20,11 @@ public DummyJavaNode(int id) {
         super(id);
     }
 
+    @Override
+    public void setImage(String image) {
+        super.setImage(image);
+    }
+
```

Comment:
Indeed, I recall it's still in main because some rules about imports create dummy nodes for ImportWrapper... Hopefully those rules may be rewritten with the newer symbol table and ImportWrapper can be nuked